Competition2018/src/main/java/competition/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -165,6 +165,10 @@ public ElevatorSubsystem(CommonLibFactory clf, XPropertyManager propMan, Electri
         if (contract.elevatorUsesTalonLimits()) {
             initializeTalonLimits();
         }
+        
+        
+        // Brutal hack since we don't trust anything
+        calibrateHere();
```

Comment:
Our sensors are not trustworthy, so the robot just calibrates on boot saying its current position is a good position.